### PR TITLE
ES6: migrated "extension"

### PR DESCRIPTION
**Pull request by herrmannplatz**

### CODE REVIEW

File: app/script/extension/GiphyRepository.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+      const _get_random_gif = (retries = 0) => {
+        if (options.retry === retries) {
+          reject(new Error(`Unable to fetch a proper gif within ${options.retry} retries`));
+        }
+
+        return this.giphy_service.get_random(options.tag)
+        .then(response => this.giphy_service.get_by_id(response.data.id))
+        .then(response => {
+          const { images } = response.data;
```
Comment (lipis) [inline app/script/extension/GiphyRepository.js]: can we remove the spaces? `{images}`

Comment (herrmannplatz) [reply app/script/extension/GiphyRepository.js]: AHHHHHHHHHHHHHHHHHHHH

File: app/script/extension/GiphyContentSizes.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.extension = z.extension || {};
+
+// Enum of different Giphy content sizes.
+z.extension.GiphyContentSizes = {
+  FIXED_HEIGHT: 'fixed_height',
```
Comment (bennycode) [inline app/script/extension/GiphyContentSizes.js]: Can get sorted 💃

File: app/script/extension/GiphyService.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  /*
+  Search GIFs for a word or phrase.
+
+  @param {Object} options
+  @param {string} options - query search query term or phrase
+  @param {number} [options.limit=25] - Number of results to return (maximum 100, default 25)
+  @param {number} [options.offset=0] - Results offset (defaults 0)
+  @param {string} [options.sorting='relevant'] - Relevant or recent
+  */
+  get_search(options) {
```
Comment (bennycode) [inline app/script/extension/GiphyService.js]: According to: http://stackoverflow.com/a/21317255/451634

This must become:

```javascript
@param {string} options - query search query term or phrase
@param {number} options.limit=25 - Number of results to return (maximum 100, default 25)
@param {number} options.offset=0 - Results offset (defaults 0)
@param {string} options.sorting='relevant' - Relevant or recent
```

**Pull request merged**